driver-core/src/main/com/mongodb/client/model/changestream/ChangeStreamDocument.java

```diff
@@ -430,6 +476,18 @@ public BsonDateTime getWallTime() {
         return wallTime;
     }
 
+    /**
+     * The split event.
+     *
+     * @return the split event
+     * @since 4.11
+     * @mongodb.server.release 7.0
```

Comment:
Since SplitEvent works with 6.0.9 and the documentation has been updated for it, let's change it to 6.0.9 as well

Reply:
Good catch